test/e2e/autoops/auto_ops_test.go

```diff
@@ -295,12 +295,12 @@ func TestOpsEventRateBatchWithoutTag(t *testing.T) {
 	time.Sleep(40 * time.Second)
 
 	userIDs := createUserIDs(t, 10)
-	for _, uid := range userIDs {
-		grpcRegisterEvaluationEvent(t, featureID, feature.Version, uid, feature.Variations[0].Id, "")
-	}
 	for _, uid := range userIDs[:6] {
 		registerGoalEventWithEvaluations(t, featureID, feature.Version, goalID, uid, feature.Variations[0].Id)
 	}
+	for _, uid := range userIDs {
```

Comment:
I changed the sending order of the events to ensure the linking implementation is working in the event persister.